src/common/commonutils/SshUtils.c

```diff
@@ -232,7 +231,8 @@ static int IsSshConfigIncludeSupported(OsConfigLogHandle log)
 
     if (NULL != textResult)
     {
-        if (((textPrefixLength = strlen(expectedPrefix)) + 3) < (textResultLength = strlen(textResult)))
+        textPrefixLength = strlen(expectedPrefix) + 3;
+        if (textPrefixLength < strlen(textResult))
```

Comment:
Just curious, why is this necessary/why is gets flagged by the analysis?

Reply:
The assignments to `textResultLength` and `textPrefixLength` are both not used outside the `if` condition, and that gets flagged as a dead store (a store that is never read).

I opted to remove the `textResultLength` variables as `strlen(textResult)` seems equally readable.
For `textPrefixLength` I found assigning `strlen(expectedPrefix) + 3` to the variable does add some clarity.